### DISCUSSION TITLE
Switch doesn't seem to want to render?

### DISCUSSION THREAD

**Discussion by riceboyler** [cschroeter/park-ui#238] [General]
First, a simple repro - https://stackblitz.com/edit/github-ttm5ka?file=src%2Fapp%2Fpage.tsx

I've been trying this in a new project where I'm moving away from Chakra-UI because I want to use RSCs as much as possible and I need to shrink our bundle (considerably). It's not working in my repo, so I decided to basically fork the example and add a tab for a Switch and that's what you see above. If you go to the Switch tab, you'll see it renders the label but not the actual Switch control. 

I'm sure the answer is easy and it's just something I'm missing, but it feels like it should be working. (FWIW, in the StackBlitz, I did run `npm run prepare` after adding the code for Switch just in case the recipe needed to be updated.)

Can anybody lead me down the proper path?

**Comment by riceboyler** [comment]
🤦  So, it appears that it doesn't like the `sm` or `lg` sizes for some reason. If I just pass through `size="md"`, it renders just fine. Is that expected behavior?

**Comment by cschroeter** [comment]
@riceboyler 

Thanks for taking your time to create this discussion / issue. Turns out this was actually a bug and it is solved in `0.35.2`

Cheers
